logging/README.md

```diff
@@ -0,0 +1,6 @@
+# Purpose
+Initialize logging (done by blitz4j) through log4j.properties and through configuration in configuration file mapped to key `logging`
```

Comment:
Maybe mention that slf4j is used as api. Fyi, I have a local branch in which I am replacing blitz4j/log4j1 with log4j2, keeping the slf4j api.